proxy.js

```diff
@@ -14,11 +14,18 @@ exports.callEndpoint = function(appId, appSecret, endpoint, query, token, callba
   };
 
   request(options, function (err, response, body) {
-    if (err || response.statusCode != 200) {
+    if (err) {
       var errMsg = err && err.message ? ' : ' + err.message : '';
-      callback(new Error('Error while trying to call platform endpoint ' + endpoint + errMsg));
+      createError(errMsg);
       return;
     }
+
+    if (response.statusCode != 200) {
+      var statusCodeMsg = 'the response status code is' + response.statusCode;
```

Comment:
Use the same format as the error message:
`Error while trying to call platform endpoint ${endpoint}. Status code: ${response.statusCode}`